addons/udes_stock/models/stock_picking.py

```diff
@@ -907,41 +907,44 @@ def _requires_backorder(self, mls):
                 return True
         return False
 
-    def _backorder_movelines(self, mls=None):
-        """ Creates a backorder pick from self (expects a singleton)
+    def _backorder_movelines(self, mls=None, picking=None):
```

Comment:
nit: Can we name it `dest_picking` or something that's not `picking`, to distinguish it from the `self` picking?

Reply:
Done, please review.

Reply:
The final `return` needs updating

Reply:
Changed too :)

Reply:
lgtm